tests/ci/cpu_unit_tests.yml

```diff
@@ -0,0 +1,101 @@
+# cpu_unit_tests.yml

```

Comment:
i would remove this line and add in the standard microsoft license header

Reply:
done